<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="le_message"/>
    </item>
    <item>
     <widget class="QPushButton" name="pb_sendDatagram">
      <property name="text">
       <string>Отправить датаграмму</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="te_receivedData"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
